server/odc-service/src/main/java/com/oceanbase/odc/service/flow/instance/FlowApprovalInstance.java

```diff
@@ -138,43 +146,51 @@ public FlowApprovalInstance(@NonNull Long organizationId, @NonNull Long flowInst
         this.autoApprove = autoApprove;
         this.waitForConfirm = waitForConfirm;
         this.externalApprovalId = null;
+        this.userTaskInstanceCandidateRepository = userTaskInstanceCandidateRepository;
         alloc();
-        create();
-        Verify.notNull(getId(), "id");
-        Verify.notNull(getCreateTime(), "CreateTime");
-        Verify.notNull(getUpdateTime(), "UpdateTime");
     }
 
     @Override
     public FlowableElementType getCoreFlowableElementType() {
         return FlowableElementType.USER_TASK;
     }
 
+    public static List<FlowApprovalInstance> batchCreate(List<FlowApprovalInstance> instances,
```

Comment:
we need to do tow operations when we attempt to create multi flow approval instances:
1. create multi flow approval instances
2. create multi flow candidate instances
so that we need to create a method to do this to make caller do not know the detail of this creation(two steps).